installer/rpm/algorand/algorand-centos-stream8.repo

```diff
@@ -0,0 +1,6 @@
+[algorand]
+name=Algorand-centos-stream8
```

Comment:
Does this naming style mean that if you want to install it, you'll have to install it as e.g. `dnf install algorand-centos-stream8`? Curious how we tested the repo setup.

Reply:
dnf-automatic is a dependency for the algorand rpm package.

I copied the rpm file locally and ran the installation on a centos stream 8 system. 

Here are the commands I used to test installation of the algorand package on centos stream 8
  117  sudo rpm -i algorand-centos-stream8-2.7-1.x86_64.rpm 
  118  ps -eafl | grep algod
  119  systemclt start algorand
  120  systemctl start algorand
  121  sudo systemctl start algorand
  122  sudo systemctl status algorand
  123  /usr/bin/goal node status -d /var/lib/algorand/
  124  rpm -qa | grep algorand
  125  sudo systemctl status algorand
  126  sudo systemctl stop algorand
  127  sudo systemctl status algorand
  128  rpm -qa | grep algorand
  129  rpm -e algorand-centos-stream8-2.7-1.x86_64
  130  sudo rpm -e algorand-centos-stream8-2.7-1.x86_64
  131  rpm -qa | grep algorand
  132  sudo systemctl status algorand

Reply:
This tests the binary, but do we have a test of installing from a test repository? I'm curious about the package name for installation and if this works in both beta and stable.

Reply:
Tested with Gemfury test repo.